python/port/mod/os/modos.cpp

```diff
@@ -32,12 +32,19 @@ STATIC const MP_DEFINE_STR_OBJ(modos_uname_info_machine_obj, "NumWorks N0100");
 STATIC const MP_DEFINE_STR_OBJ(modos_uname_info_machine_obj, "NumWorks Simulator");
 #endif
 
+#if defined(OMEGA_USERNAME)
+STATIC const MP_DEFINE_STR_OBJ(modos_uname_info_username_obj, MP_STRINGIFY(OMEGA_USERNAME));
```

Comment:
Please use [`Ion::Username`](https://github.com/Omega-Numworks/Omega/blob/6f797833b2195a1aeaa9298dcd60dd4cbd006a96/ion/include/ion.h#L35) and make this non-constant, like by using the Ion::username function in the `modos_getlogin` function.

Reply:
https://github.com/Omega-Numworks/Omega/pull/544/commits/24f3ccf007c59eec4a352a746c87f5c55b7c10e5 resolves this